collectd-openstack/README.md

```diff
@@ -66,7 +72,7 @@ Reference for OpenStack [Monitoring](https://wiki.openstack.org/wiki/Operations/
 
 2. Copy the [sample configuration file](https://github.com/signalfx/integrations/tree/release/collectd-openstack/20-openstack.conf) for this plugin to `/etc/collectd/managed_config` directory.
 
-3. Modify the sample configuration file as described in [Configuration](#configuration), below.
+3. Modify the sample configuration file as described in [Configuration](#configuration), below. To avoid duplicate reporting, configure each project only once.
```

Comment:
I can't add a comment on step 1 cause it's not in the diff, but I would add "... files into the xyz directory on the deployment host you have identified (see Requirements and Dependencies)."

Reply:
updated